scripts/publish-release.sh

```diff
@@ -20,24 +21,9 @@ fi
 
 echo "preparing release file"
 
-mkdir $RELEASE_PATH
-mkdir $RELEASE_PATH/bin
-mkdir $RELEASE_PATH/include
-mkdir -p $RELEASE_PATH/lib/pkgconfig
-mkdir $RELEASE_PATH/misc
-
-cp parameters.json $RELEASE_PATH/misc/
-cp target/release/paramcache $RELEASE_PATH/bin/
-cp target/release/paramfetch $RELEASE_PATH/bin/
-cp target/release/libfilecoin_proofs.h $RELEASE_PATH/include/
-cp target/release/libfilecoin_proofs.a $RELEASE_PATH/lib/
-cp target/release/libfilecoin_proofs.pc $RELEASE_PATH/lib/pkgconfig
-
-pushd $RELEASE_PATH
-
-tar -czf $RELEASE_FILE ./*
+`dirname $0`/package-release.sh
 
-popd
+mv /tmp/release.tar.gz $RELEASE_FILE
```

Comment:
This should probably be a var (`TAR_FILE`, as in `package-release.sh`). As far as I can tell, this depends on these being set to the same value in both scripts. Ideally, it would be impossible for them to differ, but if we need to hard-code the relationship, let's at least document.

Reply:
Interesting idea. What do you think, @sidke?

Reply:
@laser Can you approve this if/when @sidke makes a decision, or otherwise close?

Reply:
i will start work back on getting single env var precompiled installs after getting PIP in